src/sentry/tasks/commit_context.py

```diff
@@ -46,6 +47,11 @@ def process_commit_context(
             set_current_event_project(project_id)
 
             project = Project.objects.get_from_cache(id=project_id)
+            organization: Organization = Organization.objects.get_from_cache(
+                id=project.organization_id
+            )
+            set_tag("organization.slug", organization.slug)
```

Comment:
I think this should work too:
```suggestion
            set_tag("organization.slug", project.organization.slug)
```